folio-spring-i18n/src/main/java/org/folio/spring/config/TranslationConfiguration.java

```diff
@@ -0,0 +1,29 @@
+package org.folio.spring.config;
```

Comment:
One more thing that I probably should have noticed earlier: it'd be better if this library was in a distinct package (so under `org.folio.spring.i18n` or something). Otherwise, it can cause a headache later if anyone wants to use multiple libs with overlapping packages while using the java module system, since every module is required to have a distinct package (IIRC, it works when you don't explicitly use the module system because every jar on the classpath ends up in the unnamed module, so the overlapping packages are in the same module. Once you try to explicitly use modules, they end up in a different kind of separate modules (I think they are called default modules or something))